src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx

```diff
@@ -182,7 +190,9 @@ export default function SavedCodesSubSection({
       },
       {
         header: "System Version",
-        accessorKey: "svsVersion",
+        accessorKey: measureModel?.includes("QDM")
+          ? "svsVersion"
+          : "fhirVersion",
```

Comment:
same comment regarding model check

Reply:
Was playing with, but couldn't come up with, a reasonable solution for lazy load of a Util similar to how React provides `lazy()` to lazy load a component. So for this, perhaps just a single CodesModelUtil or something like that to contain the model-specific logic that isn't component/rendering